### ISSUE TITLE
Support Target option

### ISSUE THREAD

**Issue by ZIJ** [diggerhq/digger#240]
Delta plans. Similar, or an extension to the above where -target is in effect to see only desired changes.

Seems like could be addressed by introducing support of the [target option](https://developer.hashicorp.com/terraform/cli/commands/plan#target-address) for the plan command. So you'd comment digger plan -t <target> and that'll run a targeted plan. Or maybe something similar to what [tftarget](https://github.com/future-architect/tftarget) does, sort of a "persistent target" for the entire pr; then it'll be smth like digger target <target> comment that will set the target. The latter approach would introduce an extra stateful piece though in addition to locks. @timothyclarke I'm not sure that I'm getting your idea right though

Originally suggested by @timothyclarke in #237 - now tracking separately here

**Comment by danijelmarsic2** [comment]
this would be a great feature. We were using atlantis and now switched to digger and we really miss --target option which we had with atlatnis. 
For example, this would only apply a single module in our stack, rather than all 100 of them are using in our stack:
**atlantis plan -- -target=module.our-custom-module.random_password.first_random_pass**

**Comment by frank-bee** [comment]
even apply should be possible I think.
In large stacks where several teams are working ( not good but sometimes reality ) there might be "lock situation" which can't be fixed immediately ( other team not reachable...) and you want to do a `terraform apply --target ..." to apply at least your required changes

**Comment by danijelmarsic2** [comment]
> even apply should be possible I think. In large stacks where several teams are working ( not good but sometimes reality ) there might be "lock situation" which can't be fixed immediately ( other team not reachable...) and you want to do a `terraform apply --target ..." to apply at least your required changes

Exactly. I am just stuck at this use case with digger because some team member made a typo in his module which is breaking the plan and apply of the complete stack. With --target I could apply only my module and have time to locate the colleague to fix his type because I have no access to the repo where he made an error. I hope that digger will implement --target soon.

**Comment by LucaIcaro** [comment]
still no news for this issue?

**Comment by joschi36** [comment]
The newly added OpenTofu Target & Exclude files option could be very usful aswell
https://opentofu.org/docs/intro/whats-new/#target-and-exclude-files
